recipes/reaktoro/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Work in progress: using a commit hash from GitHub, as there's no tagged release yet
+{% set version = "0.1.0" %}
+
+package:
+  name: reaktoro
+  version: {{ version }}
+
+source:
+  url: https://github.com/reaktoro/reaktoro/archive/bb85d53b014fc8ab9aa22471be4f1dd4aa9b5a81.tar.gz
+  sha256: 65364d6eb1da26fd7aba61770c4deef4a7d6f1220799c362750a3f1ac89c8f89
+
+build:
+  number: 0
+  # NOTE: Linux is temporarily skipped because it'll need GCC 7
+  # (It will be added after a feedstock is created)
+  skip: True  # [py2k or not win]
```

Comment:
You can build for ``osx`` because it is using ``clang``. Let's try ;)

Reply:
I'd rather not mix things for now, I'd like to have the feedstock with the Windows package to later tinker with it to get both Linux and macOS packages at the same time ;)